### ISSUE TITLE
Line numbers are 0

### ISSUE THREAD

**Issue by pelletier** [bugsnag/bugsnag-cocoa#36]
I installed Bugsnag Cocoa using the Pod method. It seems to work fine, but the line numbers in the reports are always zero. For example:

```
/Users/thomas/Library/Developer/Xcode/DerivedData/MyApp-daxwpuwazlwktwbeipjylrhsqjps/Build/Products/Debug/MyApp.app/Contents/MacOS/MyApp:0
+[Bugsnag notify:]

/Users/thomas/Library/Developer/Xcode/DerivedData/MyApp-daxwpuwazlwktwbeipjylrhsqjps/Build/Products/Debug/MyApp.app/Contents/MacOS/MyApp:0
-[AppDelegate applicationDidFinishLaunching:]
```

Note the `:0` at the end.

I'm using revision 3317cf20b042cdb64b8fe5ddac5ad3a2ef0427ef, XCode 6.1.1, Yosemite 10.10.1, and targeting OS X 10.10.

Do you know how what might go wrong?

**Comment by ConradIrwin** [comment]
It looks like symbolication isn't working. The most common reason for this is that dSYMs are not being uploaded. If you used the github podspec `pod 'Bugsnag', :git => "https://github.com/bugsnag/bugsnag-cocoa.git"`, this should happen automatically.

If not, there are instructions on setting up the build phase https://github.com/bugsnag/bugsnag-cocoa#without-cocoapods.

Please let me know if that doesn't fix it, and we can debug further.

**Comment by pelletier** [comment]
@ConradIrwin CocoaPods has correctly set up a build phase to upload the dSYMs (with the same code as given in the "Without CocoaPods" section). When I ran the code in my terminal (replacing the env variables), the upload seemed to have worked fine.

However, I have a small upload bandwidth, so it takes around 30 seconds to upload the dSYMs to your servers. As my exception was sent in the `applicationDidFinishLaunching:` method, it was triggered *before* the dSYMs had been fully uploaded. So I decided to trigger the exception after around a minute: everything worked as expected.

I suppose this is an edge case because the users usually receive the app *way* more than 30 seconds after it is built. Maybe add a line in the documentation about that :)
